src/utils.js

```diff
@@ -714,3 +714,7 @@ module.exports.ensureNoTrailingSlash = function(url) {
 module.exports.sleep = (ms, value) => new Promise((resolve => {
     setTimeout(resolve, ms, value);
 }));
+
+module.exports.isNullOrUndefined = function(val) {
```

Comment:
rationale: I'm super lazy and don't feel like typing this OR condition all over the place.